### PR TITLE
Shaded form elements

### PR DESCRIPTION
**Pull request by azangru**
## Type
- Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1223
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1224

## Description
Added an input with a shadow (`ShadedInput`) and a textarea with a shadow (`ShadedTextarea`) to be used in forms.

Both can be viewed in storybook (in the input and textarea stories).

### CODE REVIEW

File: src/ensembl/src/shared/components/textarea/Textarea.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ type PropsForRespondingWithData = {
 
 type OnChangeProps = PropsForRespondingWithEvents | PropsForRespondingWithData;
 
-type Props = {
+export type Props = {
```
Comment (azangru) [inline src/ensembl/src/shared/components/textarea/Textarea.tsx:39]: This is the only line that I changed in this file. The rest is prettier 😞

**Review by imransl — APPROVED** [review]
Tested locally and this looks good to me

File: src/ensembl/src/styles/_settings.scss [hunk 2]
```diff
@@ -39,6 +39,7 @@ $mustard: #cc9933;
 $green: #47d147;
 
 $global-box-shadow: $medium-light-grey;
+$form-field-shadow: inset 2px 2px 4px -2px $medium-dark-grey, inset -2px -2px 1px -2px $dark-grey;
```
Comment (msmanoj) [inline src/ensembl/src/styles/_settings.scss:42]: Should it be `inset 2px 2px 4px -2px $medium-dark-grey, inset -2px -3px 3px -2px $light-grey;` to match with the XD?

Comment (msmanoj) [reply src/ensembl/src/styles/_settings.scss:42]: Andrea is happy with the current styling.

**Review by msmanoj — APPROVED** [review]

File: src/ensembl/src/styles/_settings.scss (hunk 2 above)
Comment (azangru) [reply src/ensembl/src/styles/_settings.scss:42]: Checked during code review w/Andrea, and decided to leave the current value.

**Pull request merged**